components/tabs/stories/tabs.stories.js

```diff
@@ -81,38 +119,156 @@ export default {
 	},
 };
 
+/**
+ * Basic, default tab items should have a label for accessibility. If a label isn’t present, it must include an icon and becomes an icon-only tab item.
+ * 
+ * By default, tabs have a divider that spans across all tab items. This style works as a way to anchor them to the page. These types of tabs are best used at the top of a page, as a top-level navigation.
+ * 
+ * Tabs are horizontal by default and should be used when horizontal space is limited.
+ * 
+ */
 export const Default = TabsGroups.bind({});
 Default.args = {};
 
-// ********* VRT ONLY ********* //
-export const WithForcedColors = TabsGroups.bind({});
-WithForcedColors.tags = ["!autodocs", "!dev"];
-WithForcedColors.parameters = {
-	chromatic: {
-		forcedColors: "active",
-		modes: disableDefaultModes
-	},
-};
-
 // ********* DOCS ONLY ********* //
-export const Vertical = Template.bind({});
-Vertical.tags = ["!dev"];
+/**
+ * Vertical tabs should be used when horizontal space is more generous and when the list of sections is greater than can be presented to the user in a horizontal format. 
+ */
+export const Vertical = VerticalGroup.bind({});
 Vertical.args = {
 	orientation: "vertical",
 };
+Vertical.tags = ["!dev"];
 Vertical.parameters = {
-	chromatic: {
-		disableSnapshot: true,
-	},
+	chromatic: { disableSnapshot: true },
 };
 
-export const Overflow = Template.bind({});
-Overflow.tags = ["!dev"];
+export const VerticalRight = Template.bind({});
+VerticalRight.args = {
+	...Vertical.args, 
+	hasRightAlignedTabs: true,
+};
+VerticalRight.storyName = "Vertical right";
+VerticalRight.tags = ["!dev"];
+VerticalRight.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * When there are too many tabs to fit horizontally across the viewport, the tabs component can be displayed as a [quiet picker](/docs/components-picker--docs). The example below is non-functional.
+ * 
+ * When appropriate, alternative methods of overflowing tabs such as horizontal scrolling can be used.
+ */
+export const Overflow = OverflowGroup.bind({});
 Overflow.args = {
 	orientation: "overflow",
 };
+Overflow.tags = ["!dev"];
 Overflow.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * The medium size is the default and most frequently used option. Use the other sizes sparingly; they should be used to create a hierarchy of importance within the page.
+ */
+export const Sizing = (args, context) => Sizes({
+	Template,
+	withHeading: false,
+	withBorder: false,
+	...args,
+}, context);
+Sizing.args = {...Default.args};
```

Comment:
```suggestion
Sizing.args = Default.args;
```

No need to deconstruct this unless you want to add properties.